Neos.Flow/Classes/Mvc/DispatchComponent.php

```diff
@@ -98,12 +98,11 @@ protected function prepareActionRequest(ComponentContext $componentContext)
         $arguments = $httpRequest->getArguments();
 
         $parsedBody = $this->parseRequestBody($httpRequest);
-        if ($parsedBody !== []) {
+        $httpRequest = $httpRequest->withParsedBody($parsedBody);
```

Comment:
We always want to set the parsed body, even if it is an empty array, a simple string or even null.